adafruit_io/adafruit_io.py

```diff
@@ -218,6 +222,9 @@ def add_feed_callback(self, feed_key, callback_method):
     def remove_feed_callback(self, feed_key):
         """Removes a previously registered callback method
         from executing whenever feed_key receives new data.
+
+        If an on_message callback is registered, new messages
+        will now call on_message instead of the callback_method.
```

Comment:
Please reword this. Is on_message applicable to all feeds rather than one specifically? Maybe use the phrases "generic" or "shared" versus "feed-specific".

Reply:
@tannewt Yes -`on_message` is applicable to all feeds. I'll reword.